app/templates/components/forms/login-form.hbs

```diff
@@ -54,7 +54,7 @@
                   Or
                 </div>
                 <button type="button" class="ui facebook button" {{action 'auth' 'facebook'}}>
-                  <i class="facebook icon"></i>
+                  <i class="facebook icon" style="filter: brightness(200%)"></i>
```

Comment:
No custom CSS, please.

Reply:
I am trying to correlate the login-form.scss file but unable use facebook-icon class to change the properties of icon. Could you assist me??

Reply:
I tried to change the properties using scss file with `.facebook icon {....}` but seems not working.